sda/cmd/s3inbox/proxy.go

```diff
@@ -199,6 +200,27 @@ func (p *Proxy) allowedResponse(w http.ResponseWriter, r *http.Request, token jw
 			return
 		}
 
+		// The following block is for treating the case when the client loses connection to the server and then it reconnects to a
+		// different instance of s3inbox. For more details see https://github.com/NBISweden/BigPicture-Deployment/issues/283.
```

Comment:
```suggestion
		// different instance of s3inbox. For more details see https://github.com/NBISweden/BigPicture-Deployment/issues/283.
```
Don't link to private repositories, create/transfer the issue to this repo instead.